py-polars/src/npy.rs

```diff
@@ -14,12 +15,12 @@ use std::{mem, ptr};
 /// All elements in the array are non initialized
 ///
 /// The array is also writable from Python.
-pub unsafe fn aligned_array<T: Element>(py: Python<'_>, size: usize) -> (&PyArray1<T>, *mut T) {
```

Comment:
I made this a `AlignedVec`, since it avoids the need to return a pointer and rebuild the `AlignedVec` from it: we can just keep the `AlignedVec`.